app/src/main/res/values/arrays.xml

```diff
@@ -25,6 +25,37 @@
         <item>60</item>
     </string-array>
 
+    <string-array name="gradually_increase_volume" translatable="true" >
+        <item>Never</item>
+        <item>10 seconds</item>
+        <item>20 seconds</item>
+        <item>30 seconds</item>
+        <item>40 seconds</item>
+        <item>50 seconds</item>
+        <item>1 minute</item>
+        <item>1 minute 10 seconds</item>
+        <item>1 minute 20 seconds</item>
+        <item>1 minute 30 seconds</item>
+        <item>1 minute 40 seconds</item>
+        <item>1 minute 50 seconds</item>
+        <item>2 minutes</item>
+    </string-array>
+    <string-array name="gradually_increase_volume_values" translatable="false" >
+        <item>0</item>
+        <item>10000</item>
+        <item>20000</item>
+        <item>30000</item>
+        <item>40000</item>
+        <item>50000</item>
+        <item>60000</item>
+        <item>70000</item>
+        <item>80000</item>
+        <item>90000</item>
+        <item>100000</item>
+        <item>110000</item>
+        <item>120000</item>
+    </string-array>
```

Comment:
I think it would be easier to have `SeekBarPreference` with 10s granularity for picking the number, and it won't require these strings (each string it's a more translation to do).

Reply:
👍 Fixed in 3233b6f.

![Screenshot_20191103-224135_RadioDroid](https://user-images.githubusercontent.com/144435/68093273-c1645c80-fe8b-11e9-835c-200a40ca786f.png)